include/config/battle.h

```diff
@@ -274,6 +274,9 @@
 #define B_TRY_CATCH_TRAINER_BALL        GEN_LATEST // In Gen4+, trying to catch a Trainer's Pokémon does not consume the Poké Ball.
 #define B_SLEEP_CLAUSE                  FALSE      // Enables Sleep Clause all the time in every case, overriding B_FLAG_SLEEP_CLAUSE. Use that for modularity.
 
+#define B_NUM_BEEPS_GEN_LOCK            TRUE       // Locks the number of times the "low health" beep loops to 4 or infinite. GEN_LATEST means it loops 4 times, any other GEN_* or value means it's infinite. Set this to FALSE to enable non-vanilla values, ie 0 for off, or any other number.
```

Comment:
You really don't need this true/false here. It can just be one config since its always going to be either infinite or 4

Reply:
context: https://discord.com/channels/419213663107416084/1343308120369135726/1349072219342377102